x/model/keeper/msg_server_model.go

```diff
@@ -195,5 +200,31 @@ func (k msgServer) DeleteModel(goCtx context.Context, msg *types.MsgDeleteModel)
 	// remove product from VendorProducts
 	k.RemoveVendorProduct(ctx, msg.Vid, msg.Pid)
 
+	modelVersions, err := k.ModelVersions(goCtx, &types.QueryGetModelVersionsRequest{
```

Comment:
I think it will be cleaner to use `k.GetModelVersions()`, because this code fragment will not differ from others in approach

Reply:
@DenisRybas ah, I missed this function, thanks!